### DISCUSSION TITLE
golang debugging support

### DISCUSSION THREAD

**Discussion by ydevincenzi** [microsoft/vscode-docker#3228] [Ideas]
Hi,
Is it possible to support debugging for golang containerized applications ?

**Comment by bwateratmsft** [comment]
It's not officially supported at the moment, so I created #3229 for that. It *should* be possible with the Go extension since it supports remote debugging.

The way the Docker extension does debugging for .NET, Node, and Python is to convert the `docker` debug configuration into `netcore`, `node`, etc. So it really is just a shortcut, which means that it should be possible to construct a launch configuration for Go that works end-to-end.

**Reply by ydevincenzi** [reply]
Thank you brandon. It's a nice news
